Header header
string[] joint_names
float64[] positions
float64[] velocities
float64[] kps
float64[] kds
float64[] effort